climada/entity/exposures/base.py

```diff
@@ -34,7 +34,7 @@
 from climada.entity.tag import Tag
 import climada.util.hdf5_handler as hdf5
 from climada.util.constants import ONE_LAT_KM, DEF_CRS
-import climada.util.coordinates as co
+import climada.util.coordinates as u_coord
```

Comment:
Following another convention I found in some of CLIMADA's modules, I switched to the naming scheme with a prefix `u_` for submodules of `climada.util` which is short while being recognizable. What do you think? Would you rather go for longer names, like `util_coord`?

Reply:
u_coord looks good to me. It is I think quickly clear what it is. Especially if we take the habit to use 'u_' to denote methods imported from `climada.util`.